packages/fxa-auth-server/README.md

```diff
@@ -147,6 +147,84 @@ This saves the HTML template into `/templates`. Then make changes to the `.txt`
+
+#### MJML Feature Flag for Testing
+
+The auth-server has an `mjml` configuration value that designates which email templates have been converted to the new stack ( `mjml.templates` array) and controls which user emails will receive those templates ( `mjml.enabledEmailAddress` regex). The auth-server exposes a feature flag method to check these values and send the correct rendered template accordingly (old or new) which allows for testing across environments.
+
+This flag and logic around it can be removed, as well as old templates and documentation, once all templates have been converted, verified by QA, and tested in production.
+
+#### MJML and EJS
+
+HTML email has a lot of quirks - MJML shifts the burden of maintaining solutions for these off of FxA engineers. Furthermore, using EJS with MJML helps us include more logic in templates thereby making conditional rendering possible without creating any additional helper methods. A small example of how template variables are being passed down from the mailer object and consumed in the templates is shown below:
+
+```js
+const templateValues = {
+  buttonText: 'Sync another device',
+};
+const localized = this.templates.render(
+  message.template, // template name
+  message.layout || 'fxa', // template layout
+  templateValues
+);
+const button = `
+  <mj-include path="./lib/senders/emails/css/button/index.css" type="css" css-inline="inline" />
+  <mj-section>
+    <mj-column>
+      <mj-button css-class="primary-button"><%= buttonText %></mj-button>
+    </mj-column>
+  </mj-section>
+`;
+```
+
+The emails for which the mjml feature flag is enabled can be rendered to disk using the `yarn write-emails` command.
+
+#### Styles
+
+Another advantage of using MJML for emails is that it inlines styles with the compliled HTML elements making emails compatible with all the mail clients. Currently, we are using `scss` stylesheets which get compiled down to css and are included in the mjml templates. We are maintaining two kinds of stylesheets for our emails, a global stylesheet which contains variables and shared classes, and template-specific stylesheet for the styles scoped to the respective template/partial. Although there are lot of benefits of using stylesheets with MJML, one caveat that we'd like to highlight is the inevitible use of `!important` syntax with some of the styles. While compiling down the templates, mjml internally adds some default styles to the html elements, so if we were to add our custom styles on top of it we may have to use `!important` with them to override the default ones.  
```

Comment:
My only nitpick here is maybe this:
> We are maintaining two kinds of stylesheets for our emails, a global stylesheet which contains variables and shared classes, and template-specific stylesheet for the styles scoped to the respective template/partial.

I think this may quickly become outdated (I think it may already be, with you extracting the rtl styles in the other PR?). Maybe say:

> We are maintaining shared stylesheets with common styles and variables, as well as template-specific stylesheet for the styles scoped to the respective template/partial.